### PR TITLE
Restyle license search in Locator Popup

### LINKED ISSUE #2266: Improve styling of license multi select in locate popup
**Describe the bug**
The styling of the license multi select in locate popup is inconsistent with other places.

**To Reproduce**
Open locate popup and select a license.

**Expected behavior**
The expected behavior is to see the licenses with the clear icon rendered inside of a MuiChip.

**Screenshots**
Current state:
Screenshot 2023-10-11 at 13 50 44
Target (styling like the filters in filter multi select):
Screenshot 2023-10-17 at 13 01 55

**Desktop (please complete the following information):**

- OS: all

### PR DESCRIPTION
**Pull request by meretp**
### Summary of changes
image
### Context and reason for change

Fixes #2266

### How can the changes be tested

Only style changes, green CI shows that the functionality to filter for licenses didn't break.

### CODE REVIEW

File: src/Frontend/Components/LocatorPopup/LocatorPopup.tsx
Comment (mstykow) [inline src/Frontend/Components/LocatorPopup/LocatorPopup.tsx:1]: the height of the license input field jumps, even when i select only a single license. let's ensure that at least one row of chips fits without any need for height changes of the wrapping input field.

https://github.com/opossum-tool/OpossumUI/assets/46091730/7a8009bc-be7e-4121-953d-6b9c9aa03b99

**Review by mstykow — CHANGES_REQUESTED** [review]

File: src/Frontend/Components/LocatorPopup/LocatorPopup.tsx [hunk 1]
```diff
@@ -36,6 +37,14 @@ const classes = {
     marginTop: '8px',
   },
   autocomplete: { marginTop: '12px' },
+  autocompleteInput: {
+    '& .MuiOutlinedInput-root': {
+      borderRadius: '2px',
+    },
+    '& .MuiInputBase-input': {
+      height: '25px',
```
Comment (mstykow) [inline src/Frontend/Components/LocatorPopup/LocatorPopup.tsx:45]: in design circles it is common to make all sizes divisible by 4. only exception usually is 2px.

Comment (mstykow) [reply src/Frontend/Components/LocatorPopup/LocatorPopup.tsx:45]: also, the label/placeholder is now no longer vertically centered.

image

**Review by mstykow — CHANGES_REQUESTED** [review]

File: src/Frontend/Components/LocatorPopup/LocatorPopup.tsx (hunk 1 above)
Comment (meretp) [reply src/Frontend/Components/LocatorPopup/LocatorPopup.tsx]: I tried my best to align this vertically but it seems to me that this is also not centered if I would not set the height. Decreasing the padding for the bottom seems to solve it but feels a bit hacky. If you have another suggestion how to fix it I'm happy to hear it.

**Review by mstykow — APPROVED** [review]

**Pull request merged**